Dockerfile.adm-webhook

```diff
@@ -0,0 +1,20 @@
+# Build stage
+ARG goversion
+FROM golang:${goversion}-alpine as builder
+RUN apk add build-base git mercurial
+# Download modules in a separate step for quicker builds when deps haven't changed
+ADD go.mod /spire/go.mod
+ADD go.sum /spire/go.sum
```

Comment:
There are some side effects to making this a new module, particularly since it depends on code from `github.com/spiffe/spire` (protobuf definitions, logging utilities, etc). basically the spire module will become a dependency, at a specific version.

As far as bloat to the build container, the nice thing about doing the module download in a separate step is that the modules will only be downloaded once for building all of the images (server, agent, and now this) so it won't actually result in any more bloat than will normally be produced.

Reply:
Aaah yes, alrighty then!